webapp/src/user.tsx

```diff
@@ -77,8 +76,25 @@ class AccountPanel extends sui.UIElement<AccountPanelProps, {}> {
         auth.logout();
     }
 
+    handleDeleteAccountClick = async () => {
+        const result = await core.confirmAsync({
+            header: lf("Delete Account"),
+            body: lf("You are about to delete your account. You can't undo this. Are you sure?"),
+            agreeClass: "red",
+            agreeIcon: "delete",
+            agreeLbl: lf("Delete my account"),
+        });
+        if (result) {
```

Comment:
for this kind of really deadly action, we use to ask it a second time in TouchDevelop. And github asks to actually write a piece of text as well so you can't "butt-delete" your account.

Reply:
For sure. Plan is for the user to have to type the name of their account to enable the delete button.